rust-bins/Cargo.toml

```diff
@@ -41,7 +41,7 @@ anyhow = "1.0"
 
 [dependencies.keygen_bls]
 path = "../rust-src/keygen_bls"
-version = "0"
+version = "2.0.0"
```

Comment:
For backwards compatibility we also need to make sure that the new version of the tool supports key recovery using the old method.

The keygen tool should have a `--v1` or similar flag that supports key recovery with the legacy keygen_bls version.